cmake/Modules/opm-simulators-prereqs.cmake

```diff
@@ -19,7 +19,8 @@ set (opm-simulators_DEPS
 	# DUNE prerequisites
 	"dune-common REQUIRED;
 	dune-istl REQUIRED"
-	# OPM dependency
+	"ERTPython"
```

Comment:
i see nothing conditional in the macro. do you plan to put the if's outside or is this to be required?